lib/elixir/lib/record.ex

```diff
@@ -36,6 +36,10 @@ defmodule Record do
 
   """
 
+  @type type :: :defrecord | :defrecordp
+  @type name :: atom
+  @type tag :: atom
+
```

Comment:
Not clear whether it should be desired to have all of some to be either public or private